Python/optimizer_analysis.c

```diff
@@ -588,16 +588,15 @@ remove_globals(_PyInterpreterFrame *frame, _PyUOpInstruction *buffer,
     INST->oparg = ARG;            \
     INST->operand = OPERAND;
 
+#define OUT_OF_SPACE_IF_NULL(EXPR) \
```

Comment:
Can you wrap this in a `do { ... } while (0)` 
It probably won't matter, but it is good practice just in case someone write this:
```C
if OUT_OF_SPACE_IF_NULL(expr) else ...
```